### PR TITLE
Add php-fpm exporter

### PR DESCRIPTION
**Pull request by kubicgruenfeld**
#### Pull Request (PR) description

This PR adds the php-fpm exporter from https://github.com/hipages/php-fpm_exporter

#### This Pull Request (PR) fixes the following issues
n/a

### CODE REVIEW

**Comment by kubicgruenfeld** [comment]
Puppet 6 Tests on Ubuntu 16.04 and 18.04 are failing because of https://github.com/voxpupuli/beaker-hostgenerator/pull/244

**Comment by kubicgruenfeld** [comment]
I don't know why Archlinux is failing. But it's not related to my changes.
CC @bastelfreak

File: manifests/php_fpm_exporter.pp [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    default => undef,
+  }
+
+  $options = "server --phpfpm.scrape-uri '${scrape_uri}' ${extra_options}"
+
+  if $install_method == 'url' {
+    # Not a big fan of copypasting but prometheus::daemon takes for granted
+    # a specific path embedded in the prometheus *_exporter tarball, which
+    # php-fpm_exporter lacks currently as of version 2.0.4
+    # TODO: patch prometheus::daemon to support custom extract directories
```
Comment (baurmatt) [inline manifests/php_fpm_exporter.pp:100]: Is there an open issue for this? If not, please open one.

**Review by baurmatt — APPROVED** [review]
LGTM. Please check the comment I've added.

(I will keep the PR unmerged for now because @kubicgruenfeld and I are colleagues. Please add an additional approve if you think this can be merged :))

File: manifests/php_fpm_exporter.pp (hunk 1 above)
Comment (kubicgruenfeld) [reply manifests/php_fpm_exporter.pp:100]: https://github.com/voxpupuli/puppet-prometheus/issues/606

**Pull request merged**